packages/components/src/components/text/text.jsx

```diff
@@ -1,27 +1,27 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
+const Text = ({ children, ...props }) => {
     function setStyle() {
         const style = {
-            '--text-size': `var(--text-size-${size || 's'})`,
-            '--text-color': `var(--text-${color || 'general'})`,
-            '--text-lh': `var(--text-lh-${lineHeight || (as === 'p' ? 'm' : 'xs')})`,
-            '--text-weight': `var(--text-weight-${weight || 'normal'})`,
+            '--text-size': `var(--text-size-${props.size || 's'})`,
+            '--text-color': `var(--text-${props.color || 'general'})`,
+            '--text-lh': `var(--text-lh-${props.lineHeight || 'm'})`,
```

Comment:
Can we name `lineHeight` to `line_height` as per styling guide (now that no one is using this component yet)?

Reply:
Sure, but I have refactore it in all core package :D